rs/http_utils/BUILD.bazel

```
@@ -37,3 +38,14 @@ rust_test(
     crate = ":http_utils",
     deps = DEPENDENCIES + DEV_DEPENDENCIES,
 )
+
+rust_test(
+    name = "http_utils_long_test",
+    srcs = glob(["tests/**/*.rs"]),
+    env = MAINNET_ENV,
+    tags = [
+        "long_test",
```

Let's document the reason for this tag:
```suggestion
        "long_test", # this test doesn't necessarily take long but it downloads a big image from AWS S3 so to save bandwidth we don't want to do that for every update to every PR.
```

---

I moved it to use the `http://download.proxy-global.dfinity.network:8080/ic/{}/guest-os/update-img/update-img.tar.zst` so that we avoid AWS if possible. Effectively it should be the same